commit	5132ca58a0a3ada1aee99732fd30d53ebe5af984
author	saiarcot895 <saiarcot895@gmail.com>	Mon Aug 22 18:41:54 2016
committer	Commit bot <commit-bot@chromium.org>	Mon Aug 22 18:44:13 2016
tree	b4783f1c90f2fdb001843ad83463e260de9b292c
parent	da7b21cac1ded0c70cb330e4f76a1f7915e38ccf

Fix compiling with CUPS 2.2 backend.

cupsGetPPD() and cupsGetPPD2() have been moved into the cups/ppd.h
header starting with CUPS 2.2. This header file now needs to be included
to use these functions. The header file also exists on older versions of
CUPS, so a compile error will not occur.

R=skau@chromium.org, thestig@chromium.org

Review-Url: https://codereview.chromium.org/2258333004
Cr-Commit-Position: refs/heads/master@{#413484}

diff --git printing/backend/cups_helper.h printing/backend/cups_helper.h
index fd7601d..b72903f 100644
--- printing/backend/cups_helper.h
+++ printing/backend/cups_helper.h

@@ -6,6 +6,7 @@
 #define PRINTING_BACKEND_CUPS_HELPER_H_
 
 #include <cups/cups.h>
+#include <cups/ppd.h>
 
 #include "base/strings/string_piece.h"
 #include "printing/printing_export.h"
